packages/styles/src/utilities/_variables.scss

```diff
@@ -31,6 +31,144 @@ $position-values: (
 );
 
 $utilities: (
+  'border': (
+    responsive: true,
+    property: border,
+    class: border,
+    values: (
+      null: var(--post-border-width) var(--post-border-style)
+        rgba(var(--post-border-color-base), var(--post-border-opacity)),
```

Comment:
This is good! Please use the same logic for the `border-top`,  `border-bottom`,  `border-start`, and  `border-end` utilities